clang/test/Driver/sycl-no-gpu-rdc-opaque-pointers.cpp

```diff
@@ -0,0 +1,18 @@
+///
+/// Checks that the SYCL libraries modules are first linked with opaque-pointers, 
+/// when no GPU Relocatable Device Code mode is used.
+///
+
+// UNSUPPORTED: system-windows
+
+/// FIXME: remove once opaque pointers are supported for all SYCL targets (i.e., SPIR-V)
```

Comment:
@mdtoguchi, @sarnex, I think SPIR-V supports opaque pointers already.
I'm not sure I understand what exactly has to be removed. Could you check if we need to make any additional changes to this patch, please?

Reply:
Sorry for late response. Sounds good, thanks.